help/develop/function_results_cache.md

```diff
@@ -0,0 +1,83 @@
+# Caching function results
+
+Datagrok can cache the results of functions on both client and server sides.
+Only those functions that have one or more output parameters can be cached.
```

Comment:
This is not necessary. If you say that Datagrok caches results, then it's clear that if a function has no results, it won't be cached.